storages/jsonl-storage/tests/jsonl_stroage_sample.rs

```diff
@@ -0,0 +1,273 @@
+                Value::Str("Ms. Lucille Mann Jr.".to_owned()),
+                Value::Bytea(bytea("acdBDdA1A5")),
+                Value::Inet(IpAddr::V4(Ipv4Addr::new(61, 202, 236, 23))),
+                Value::Date(date!("2022-12-09")),
+                Value::Timestamp(date!("2022-12-03T23:26:08")),
+                Value::Time(date!("14:16:09")),
+                Value::Interval(Interval::seconds(-(30 * 60 + 11))),
+                Value::Uuid(parse_uuid("93e6722e-7537-4ea5-9039-5917c76463b6")),
+                m(r#"{"age": 39, "city": "Lake Viola"}"#),
+                l(r#"["orange", "lavender", "teal"]"#),
+            ],
+            vec![
+                Value::Bool(true),
+                Value::I8(27),
+                Value::I16(15926),
+                Value::I32(1001101077),
+                Value::I64(8343487625208526000),
+                Value::U8(169),
+                Value::Str("Gertrude Lang".to_owned()),
+                Value::Bytea(bytea("5EA2bECF9b")),
+                Value::Inet(IpAddr::V4(Ipv4Addr::new(63, 164, 84, 111))),
+                Value::Date(date!("2023-01-04")),
+                Value::Timestamp(date!("2022-04-18T08:12:12")),
+                Value::Time(date!("08:09:18")),
+                Value::Interval(Interval::minutes(5)),
+                Value::Uuid(parse_uuid("311ac726-8ab1-4f08-9e90-6e07b736d227")),
+                m(r#"{"age": 83, "city": "Port Thalia"}"#),
+                l(r#"["olive", "olive", "cyan"]"#),
+            ],
+            vec![
+                Value::Bool(false),
+                Value::I8(27),
+                Value::I16(5185),
+                Value::I32(863367621),
+                Value::I64(2836074866821069300),
+                Value::U8(246),
+                Value::Str("Lyle Jaskolski".to_owned()),
+                Value::Bytea(bytea("FafAe1C0f0")),
+                Value::Inet(IpAddr::V4(Ipv4Addr::new(162, 30, 187, 209))),
+                Value::Date(date!("2023-02-15")),
+                Value::Timestamp(date!("2022-07-29T22:22:18")),
+                Value::Time(date!("18:52:46")),
+                Value::Interval(Interval::minutes(84 * 60 + 30)),
+                Value::Uuid(parse_uuid("a4aa6524-e49e-4d71-a2e1-20763c0a3344")),
+                m(r#"{"age": 56, "city": "Lemkeboro"}"#),
+                l(r#"["maroon", "magenta", "azure"]"#),
+            ],
+        ],
+    };
+    assert_eq!(actual, &expected);
+
+    let actual = glue.execute("SELECT * FROM WrongFormat");
+    let expected = Err(ValueError::InvalidJsonString("{".to_owned()).into());
+
+    assert_eq!(actual, expected);
+
+    let actual = glue.execute("SELECT * FROM WrongSchema");
+    let expected = Err(Error::Schema(SchemaParseError::CannotParseDDL));
+
+    assert_eq!(actual, expected);
+
+    test(
+        glue.execute("SELECT * FROM UnsupportedPrimaryKey")
+            .map(|mut payloads| payloads.remove(0)),
+        Err(Error::StorageMsg("primary key is not supported".to_owned())),
```

Comment:
## Issue
```sql
CREATE TABLE Item (id int PRIMARY KEY, name text);
INSERT INTO Item VALUES (2, 'b');
INSERT INTO Item VALUES (1, 'a');

Above query will get `duplicated key` error cuz INSERT Statement was trying to use id as primary key but internally primary key is fixed to `lineNumber`.
Line1: {id: 2, name: 'b'} already existed so that it threw `duplicated key` error.

## Todo
Throw PrimaryKeyNotSupported Error with `validata_primary_key` at `fetch_schema` and `insert_schema`.

## Another Issue
But `generate_store_tests` includes `PK tests` as well so that we might split PK test (or feature)